wgpu-native/src/command/render.rs

```diff
@@ -336,7 +337,7 @@ pub extern "C" fn wgpu_render_pass_set_vertex_buffers(
             .buffers
             .use_extend(&*buffer_guard, id, (), BufferUsage::VERTEX)
             .unwrap();
-        vbs.total_size = buffer.size - offset;
+        vbs.total_size = buffer.size - (offset + start_slot as u64);
```

Comment:
The starting slot is not in any way related to the buffer offset. it affects the position in `pass.vertex_state.inputs` that we start in.

Reply:
Oh I see. Pushed a new version.